### PR TITLE
Update code-of-conduct.rst

### PR DESCRIPTION
**Pull request by dac514**
Change some english
Move "The use of sexualized language or imagery" down the list
Add another line of unnaceptable behaviours

### CODE REVIEW

File: code-of-conduct.rst [hunk 1]
```diff
@@ -33,36 +33,36 @@ Encouraged Behaviour
...
 
 Unacceptable Behaviour
 ----------------------
 
 Examples of unacceptable behaviour by participants include:
 
-- The use of sexualized language or imagery
 - Personal attacks
-- Trolling or insulting/derogatory comments
+- Trolling, personal insultins, or derogatory comments
```
Comment (ramsey) [inline code-of-conduct.rst]: Typo? Should be "insults?"

Comment (dac514) [inline code-of-conduct.rst]: @ramsey Whoops, fixed in next commit. Thanks!

**Comment by dshafik** [comment]
:+1: on this, very nice updates to language.

**Comment by dac514** [comment]
Thanks @dshafik

**Comment by derickr** [comment]
I think this has now been addressed through merging #21 and #28. Please have a look again in case I missed something.

**Comment by dac514** [comment]
Ok.

**Pull request closed without merging**